demo/Dockerfile

```diff
@@ -46,3 +46,12 @@ COPY run-demo.sh /usr/local/bin/run-demo.sh
 ENV JAVA_OPTS -Djenkins.install.state=INITIAL_SECURITY_SETUP
 
 CMD /usr/local/bin/run-demo.sh
+
+RUN \
+    apt-get update && \
+    apt-get install -y apt-transport-https ca-certificates && \
+    apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D && \
+    echo "deb https://apt.dockerproject.org/repo debian-jessie main" > /etc/apt/sources.list.d/docker.list && \
+    apt-get update && \
+    apt-get install -y docker-engine
```

Comment:
This is all copied more-or-less from https://docs.docker.com/engine/installation/linux/debian/